src/FluentMigrator.Console/FluentMigrator.Console.csproj

```diff
@@ -25,7 +25,7 @@
     <PackageReference Include="Microsoft.Data.Sqlite" Version="6.0.4" />
     <PackageReference Include="Oracle.ManagedDataAccess" Version="12.2.1100" />
     <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.0.7" />
-    <PackageReference Include="MySql.Data" Version="8.0.28" />
+    <PackageReference Include="MySql.Data" Version="8.0.31" />
```

Comment:
I think it will better to move to [MySqlConnector](https://www.nuget.org/packages/MySqlConnector/2.2.7)